### ISSUE TITLE
[A11y] Timeline index page

### ISSUE THREAD

**Issue by Dananji** [avalonmediasystem/avalon#6420]
### Description

From the accessibility checks performed on timeline related pages in https://github.com/avalonmediasystem/avalon/issues/6314, the following issue was identified for the timeline index page.

| Success criterion | WCAG 2.1 Level | Accessibility checker | Error | Relevant HTML element |
| --- | --- | --- | --- | --- |
| 4.1.2 Name, Role, Value | Level A | SiteImprove & WAVE | Form field missing a label | Tag filter drop-down menu |

### Done Looks Like
- [ ] Add label to tag filter
